### DISCUSSION TITLE
3rd KRATOS COMMUNITY MEETING

### DISCUSSION THREAD

**Discussion by maceligueta** [KratosMultiphysics/Kratos#8580] [General]
# **3rd KRATOS COMMUNITY MEETING**

### https://unipd.zoom.us/j/81286277636  
[****** add meeting to my Google Calendar ******](https://calendar.google.com/event?action=TEMPLATE&tmeid=NnQwbGxtYzQ5MGpqcm1nNGFzY2Y3MWlqb3EgNW9kY2liY2FsYmY5bWQ1dDZlcTc5OW41cXNAZw&tmsrc=5odcibcalbf9md5t6eq799n5qs%40group.calendar.google.com)

This meeting follows the series started with the [1st Kratos Community Meeting](https://github.com/KratosMultiphysics/Kratos/discussions/8239)

## ADDRESSED TOPICS:

- Performance vs Flexibility
- (Answers to other posted questions)

### •	Post your questions, doubts, issues and suggestions as contributions to this discussion before 16th April 2021

•	Use **‘thumb up’ or upvoting** to support previously written questions. Some of the questions are ported from the previous meeting, because there was not enough time to address all of them. The most popular questions will have priority

## AGENDA for 30th March 2021 at 9.30 (CET)

▪	Welcome and introduction (5 mins)

▪	Performance vs Flexibility (10 mins)

▪	Other questions are addressed  (10 mins) 

▪	Open discussion by ‘raising hands’ system (5 mins)

@maceligueta @antonialarese @rolandwuechner

**Comment by marandra** [comment]
Idea: Add new (or repurpose existing) tests to have a "standardized" way keep track of performance improvement. It may also be useful for comparison with other software.

**Comment by antonialarese** [comment]
@KratosMultiphysics/all  
Just a reminder: next **Friday 30th March 2021 at 9.30** (CET)
3rd Kratos Community Meeting.

Link: https://unipd.zoom.us/j/81286277636

Topics:
1) Performance vs Flexibility (P. Dadvand) @pooyan-dadvand 
2) Documentation (update) (C. Roig) @roigcarlo 

Looking forward to seeing you all on Friday.

@maceligueta @antonialarese @rolandwuechner

**Reply by jginternational** [reply]
menos mal

**Comment by antonialarese** [comment]
Pdf of the slides on FLEXIBILITY vs PERFORMANCE by @pooyan-dadvand 
[KratosFlexibilityVsPerformance.pdf](https://github.com/KratosMultiphysics/Kratos/files/6404077/KratosFlexibilityVsPerformance.pdf)
